packages/x-components/src/x-modules/search-box/components/__tests__/search-input-placeholder.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { DeepPartial } from '@empathyco/x-utils';
+import { createLocalVue, mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue-global-events';
+import Vuex, { Store } from 'vuex';
+import { VueConstructor } from 'vue';
+import { SearchInputPlaceholder } from '..';
+import { isXComponent } from '../../../../components/x-component.utils';
+import { RootXStoreState } from '../../../../store/store.types';
+import { installNewXPlugin, getDataTestSelector } from '../../../../__tests__/utils';
+import { resetXSearchBoxStateWith } from './utils';
+
+let localVue: VueConstructor<Vue>;
+let store: Store<DeepPartial<RootXStoreState>>;
+
+const FakeAnimation = {
+  name: 'FakeAnimation',
+  template: '<transition v-on="$listeners"><slot /></transition>'
+};
+
+function mountFakeAnimation(): Wrapper<Vue> {
+  return mount(FakeAnimation, {
+    localVue
+  });
+}
+
+function mountSearchInputPlaceholder(options = {}): Wrapper<SearchInputPlaceholder> {
```

Comment:
To follow the same approach as other [tests](https://github.com/empathyco/x/blob/main/packages/x-components/src/x-modules/search-box/components/__tests__/search-input.spec.ts) in the project, I would encapsulate the logic of creating the local Vue, installing the x plugin, and resetting the state that is currently done in the `beforeEach` callback, in this method. 
This `mountSearchInputPlaceholder` method would instead return an object exposing the wrapper of the `SearchInputPlaceholder` component and other methods that could be needed in the tests (i.e. for emitting events).